app/models/lookup_key.rb

```diff
@@ -1,3 +1,5 @@
+require "parameters/caster"
```

Comment:
What doesn't work precisely?  It seems fine in a new Rails server in a dev env when I'm editing smart class params with various overrides and this line's commented out.